connector/src/main/scala/com/microsoft/kusto/spark/utils/KeyVaultADALAuthenticator.scala

```diff
@@ -19,7 +18,7 @@ class KeyVaultADALAuthenticator(clientId: String, clientKey: String) {
     new KeyVaultClient(createCredentials)
   }
 
-  private def createCredentials: ServiceClientCredentials = {
+  private def createCredentials: KeyVaultCredentials = {
```

Comment:
Tried it and it didn't help in Databricks - please check

Reply:
will check - although no reason not to keep it that way

Reply:
I got passed this step in Databricks. I still get an authentication error but it's due to incorrect usage